### ISSUE TITLE
Payment can't be done in cents

### ISSUE THREAD

**Issue by srimanaravind** [enupal/stripe#135]
### Description
Payment not allowed in cents using the front-end forms

![Selection_007](https://user-images.githubusercontent.com/4092725/91821866-54238280-ec7a-11ea-8cc6-03c28e9fa057.png)

Pull request: https://github.com/enupal/stripe/pull/134
### Steps to reproduce

1. Payment form -> enter an amount in cents, eg: 10.50
2.

### Additional info

- Craft version: Craft Pro 3.5.5
- PHP version: 7.2.33
- Database driver & version: MySQL 5.5.5
- Plugin version: 3.0.1
- Is [SCA](https://docs.enupal.com/stripe-payments/getting-started/sca.html) and Stripe Checkout enabled?: yes

**Comment by andrelopez** [comment]
Hi @srimanaravind we'll add support to pass the format in the next release.

**Comment by srimanaravind** [comment]
@andrelopez thanks, I already created a pull request in case you need it

**Comment by andrelopez** [comment]
Hi @srimanaravind we just released Stripe Payments v3.1.0 with support for `customAmountStep` via [options](https://docs.enupal.com/stripe-payments/templating/paymentform.html#options)

Thanks for the PR 

```twig
{{ craft.enupalStripe.paymentForm('handle', {'customAmountStep' : "0.01"}) }}
```
